src/Supportive/Console/Symfony/SymfonyOutput.php

```diff
@@ -47,4 +47,9 @@ public function isDebug(): bool
     {
         return $this->symfonyOutput->isDebug();
     }
+
+    public function writeln(string $message): void
+    {
+        $this->symfonyOutput->writeln($message);
```

Comment:
Is this really necessary? The documentation says ` 0 is considered the same as self::OUTPUT_NORMAL | self::VERBOSITY_NORMAL`
`

Reply:
refactored